source/usage-examples/findOne.rst

```diff
@@ -0,0 +1,23 @@
+===============
+Find a Document
+===============
+
+.. default-domain:: mongodb
+
+Overview
+--------
+
+You can find a single document using the ``collection.findOne()``
+method. ``findOne`` takes a query document, an optional sort order,
+and an optional projection. ``findOne`` returns a Promise that resolves
+to the first document matching the query in the specified sort order,
+which defaults to `natural sort order <https://docs.mongodb.com/manual/reference/glossary/#term-natural-order>`_.
+If no document matches the query, ``findOne`` returns a Promise that
+resolves to ``null``. Since ``findOne`` returns only one document, the
+Promise returned by this method resolves to an Object, not a Cursor.
```

Comment:
Same thing as above for the words 'Object' and 'Cursor', actually this raises an interesting concern:

Should types be capitalized? +1 @nlarew for his thoughts here.

Reply:
@mongomoe @nathan-contino-mongo Types should be capitalized since they are proper nouns (person, place, or *thing*).